.github/workflows/github-actions.yml

```diff
@@ -0,0 +1,43 @@
+# Modified for MoinMoin, copied from: https://github.com/actions/starter-workflows/blob/main/ci/python-package.yml
+
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  # push:
+  #   branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
https://github.com/borgbackup/borg/blob/master/.github/workflows/ci.yml#L78

have a look there ^^^

you can also take some other stuff from there, e.g. running the linter only once and before the real testing begins.